front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/plugins/PluginInfo.java

```diff
@@ -77,7 +77,7 @@ public void setReleaseByVersion(String version, Release release) {
   /** A singular {@code PluginInfo} release. */
   @Data
   public static class Release {
-    public static final String VERSION_PATTERN = "^[1-9]\\d*\\.\\d+\\.\\d+(?:-[a-zA-Z0-9]+)?$";
+    public static final String VERSION_PATTERN = "^[0-9]\\d*\\.\\d+\\.\\d+(?:-[a-zA-Z0-9]+)?$";
```

Comment:
Versions can start with 0, not sure why I didn't catch this.

Reply:
any reason this regex isn't just `^\\d+\\.\\d+\\.\\d+(?:-[a-zA-Z0-9]+)?$";` ?